### PR TITLE
Change to SimpleAjax extension

### PR DESCRIPTION
**Pull request by richardhj**
I forked leounglabub's extension so it is not *unmaintained* anymore.

### CODE REVIEW

**Comment by discordier** [comment]
I am undecided if we want to add a nother dependency here for the endpoint.
For Contao 4 we will then remove it again (as we will use a proper controller with a custom route then).

@MetaModels/team what do you think about this?

**Comment by richardhj** [comment]
RTM

**Comment by zonky2** [comment]
..have we discussed this on the 28.02.2017 at the MM-call?

**Comment by zonky2** [comment]
we use the PR (https://github.com/richardhj/contao-simple-ajax) from @richardhj after any small changes...

[MM-call 30.05.2017]

File: src/MetaModels/Filter/Setting/LevenstheinAjax.php [hunk 1]
```diff
@@ -26,76 +27,54 @@
...
      * @return void
      *
+     * @internal param SimpleAjax\Event\SimpleAjax $event
+     *
      * @SuppressWarnings(PHPMD.Superglobals)
      * @SuppressWarnings(PHPMD.CamelCaseVariableName)
      */
     public function handle()
     {
-        $input = \Input::getInstance();
```
Comment (richardhj) [inline src/MetaModels/Filter/Setting/LevenstheinAjax.php]: Backport

File: src/MetaModels/Filter/Setting/LevenstheinSearchSetting.php [hunk 2]
```diff
@@ -176,7 +176,7 @@ function extractLast(term) {
         .autocomplete({
             source: function(request, response) {
                 $.getJSON(
-                    "mm_lv_search.php",
```
Comment (richardhj) [inline src/MetaModels/Filter/Setting/LevenstheinSearchSetting.php:179]: Introduce service for retrieving ajax endpoint

**Review by discordier — CHANGES_REQUESTED** [review]
As discussed on Mumble on 2017-07-25, we want to get rid of the exit expressions.
@richardhj will update his simple ajax to also accept Response instances and then this is RTM.

**Comment by discordier** [comment]
This now needs a dependency on `symfony/http-foundation`.

**Comment by richardhj** [comment]
This is now a dependency of simple-ajax.

**Comment by discordier** [comment]
Yet, it also is one of this repository now, as we create a new response instance here

**Comment by zonky2** [comment]
@discordier & @richardhj - was ist den nun aus dem PR geworden?!?

**Comment by richardhj** [comment]
Gute Frage.
Ich denke wir müssen nichts mehr mergen hier.
v2.1. hat jetzt eine eigene Route bekommen btw.

**Comment by zonky2** [comment]
@discordier bitte Statement dazu - bitte fixen/mergen oder schließen...

**Comment by discordier** [comment]
@richardhj In essence this means we won't fix it for MM 2.0 at all?

**Comment by zonky2** [comment]
@richardhj @discordier ja was den nun?!?!?

**Pull request closed without merging**